app/helpers/sso_openid/application_helper.rb

```diff
@@ -32,7 +32,7 @@ def authenticate_admin!
         return true
       else
         session[:stored_location] = request.path
-        redirect_to sso_openid.auth_path(params.except("controller", "action"))
+        redirect_to sso_openid.auth_path
```

Comment:
@tmking 
lib/sso_openid/paths.rb
i believe above file has the auth_path method being called